docs/source/api/entities.md

```diff
@@ -0,0 +1,329 @@
+
+### Endpoint
+
+The main entry point to send import events is:
+Mutation.sendBulkImportEvents. The mutation takes an array of events that consist out of a type, an operation and a payload.
+
+Supported entity types:
+- PRODUCT
+- ASSORTMENT
+- FILTER
+
+Supported operations:
+- CREATE
+- UPDATE
+- REMOVE
+
+All Events follow this JSON Structure:
+
+```json
+{
+  "entity": "ENTITY TYPE",
+  "operation": "OPERATION TYPE",
+  "payload": { ... }
+}
+```
+
+Always try to send as many events at a time, so Unchained can optimize write operations.
+
+## JSON Reference
+
+### Entity Type: Product
+
+Set by unchained:
+- authorId,
+- slug history
+- _id, contentHash, created & updated if not provided
+
+```json
+{
+  "entity": "PRODUCT",
+  "operation": "CREATE",
+  "payload": {
+    "specification": {
+      "_id": "A",
+      "created": null,
+      "updated": null,
+      "sequence": "",
+      "tags": ["nice"],
+      "type": "SimpleProduct",
+      "status": "ACTIVE",
+      "published": "2020-01-01T00:00Z",
+      "commerce": {
+        "salesUnit": "ST",
+        "salesQuantityPerUnit": "1",
+        "pricing": [
+          {
+            "isTaxable": true,
+            "isNetPrice": true,
+            "countryCode": "CH",
+            "currencyCode": "CHF",
+            "amount": 10000,
+            "maxQuantity": null
+          }
+        ]
+      },
+      "warehousing": {
+        "baseUnit": "ST",
+        "sku": ""
+      },
+      "supply": {
+        "weightInGram": 0,
+        "heightInMillimeters": 0,
+        "lengthInMillimeters": 0,
+        "widthInMillimeters": 0
+      },
+      "proxy": {
+        "assignments": [
+          {
+            "vector": {
+              "color": "red"
+            },
+            "productId": "B"
+          }
+        ]
+      },
+      "plan": {
+        "billingInterval": "daily",
+        "billingIntervalCount": 1,
+        "usageCalculationType": "metered",
+        "trialInterval": "daily",
+        "trialIntervalCount": 1
+      },
+      "bundleItems": [
+        {
+          "productId": "c",
+          "quantity": 1,
+          "configuration": [
+            {
+              "key": "greeting",
+              "value": "For my Darling"
+            }
+          ]
+        }
+      ],
+      "meta": {},
+      "content": {
+        "de": {
+          "created": null,
+          "updated": null,
+          "vendor": "Herstellername",
+          "brand": "Marke",
+          "title": "Produktname",
+          "slug": "produktname",
+          "subtitle": "Short description",
+          "description": "Long description",
+          "labels": [
+            "Neu"
+          ]
+        }
+      }
+    },
+    "media": [
+      {
+        "_id": null,
+        "created": null,
+        "updated": null,
+        "asset": {
+          "_id": null,
+          "contentHash": null,
+          "url": "https://www.story.one/media/images/poop-4108423_1920.width-1600.format-jpeg.jpg"
```

Comment:
There should be the possibility to have media files bound to a language. E.g. A product-image might look different in German than in English. But it should be trivial to select the right one. Maybe add an optional field `language`?

Reply:
This is totally unsupported by unchained (and you would have to solve it by introduction of "lang" tags or something) at the moment so I will skip this

Reply:
Right, tags can solve this.